gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,379 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF (SELECT c.oid IS NOT NULL
+		   AND a.attrelid IS NULL
+		FROM pg_class c
+		JOIN pg_namespace n ON n.oid = c.relnamespace
+		LEFT JOIN pg_attribute a ON a.attrelid = c.oid AND
+		                            a.attname = 'tablespace_location'
+		WHERE c.relname = 'db_files_current' AND
+		      n.nspname = 'arenadata_toolkit')
+	THEN
+		ALTER TABLE arenadata_toolkit.db_files_current
+		ADD COLUMN tablespace_location TEXT;
+	END IF;
+END $$;
+
+DO $$
+BEGIN
+	IF EXISTS (SELECT 1
+			   FROM pg_tables
+			   WHERE schemaname LIKE 'pg_temp%' AND
+			         tablename = 'db_files_current')
+	THEN
+		DROP TABLE pg_temp.db_files_current;
+	END IF;
+END $$;
+
+DO $$
+BEGIN
+	IF (SELECT c.oid IS NOT NULL
+		   AND a.attrelid IS NULL
+		FROM pg_class c
+		JOIN pg_namespace n ON n.oid = c.relnamespace
+		LEFT JOIN pg_attribute a ON a.attrelid = c.oid AND
+		                            a.attname = 'tablespace_location'
+		WHERE c.relname = '__db_files_current' AND
+		      n.nspname = 'arenadata_toolkit')
+	THEN
+		CREATE OR REPLACE VIEW arenadata_toolkit.__db_files_current
+		AS
+		SELECT
+			c.oid AS oid,
+			c.relname AS table_name,
+			n.nspname AS table_schema,
+			c.relkind AS type,
+			c.relstorage AS storage,
+			d.datname AS table_database,
+			t.spcname AS table_tablespace,
+			dbf.segindex AS content,
+			dbf.segment_preferred_role AS segment_preferred_role,
+			dbf.hostname AS hostname,
+			dbf.address AS address,
+			dbf.full_path AS file,
+			dbf.size AS file_size,
+			dbf.modified_dttm AS modifiedtime,
+			dbf.changed_dttm AS changedtime,
+			CASE
+				WHEN 'pg_default' = t.spcname THEN gpconf.datadir || '/base'
+				WHEN 'pg_global' = t.spcname THEN gpconf.datadir || '/global'
+				ELSE (SELECT tblspc_loc
+					  FROM gp_tablespace_segment_location(t.oid)
```

Comment:
May we move this to arenadata_toolkit.__db_segment_files because it already has pg_tablespace and gp_segment_configuration?